Mithril update 2024-01-10 (#318)
IntersectMBO/cardano-updates: blog/2024-01-10-mithril.md

---
title: Mithril Team Update
slug: 2024-01-10-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team worked on implementing the client in the explorer in order to verify the certificates directly from the browser. The team also fixed the configuration of the signer node service which was responsible for some signers sometimes not registering their keys following a restart. Additionally, the team kept implementing a mock aggregator to strengthen Mithril client tests in WASM, kept working on threat modeling and risk analysis for P2P networking, and progressed with the upgrade of the devnet to handle the Conway era.

Reviewer: ```suggestion
This week, the Mithril team worked on implementing the client in the explorer to enable direct verification of certificates from the browser. They also addressed the configuration issue in the signer node service, which occasionally resulted in signers not registering their keys after a restart. Additionally, the team kept implementing a mock aggregator to strengthen Mithril client tests in WASM, continued working on threat modeling and risk analysis for P2P networking, and progressed with the devnet upgrade to support the Conway era.
```

> Finally, they worked on enhancing the node communications between Mithril and Cardano, and they completed the upgrade of the Cardano node topology used in the Mithril networks' infrastructure.

Reviewer: ```suggestion
Finally, they worked on enhancing node communications between Mithril and Cardano and completed the upgrade of the Cardano node topology used in the Mithril networks' infrastructure.
```

## Low level overview

> - Published of a dev blog post [**Mithril signer service new configuration**](https://mithril.network/doc/dev-blog/2024/01/03/signer-service-new-configuration/)

Reviewer: ```suggestion
- Published a development blog post [**Mithril signer service new configuration**](https://mithril.network/doc/dev-blog/2024/01/03/signer-service-new-configuration/)
```
